examples/bpi-monolith-test-suite/bpi/components/remotes/test.ts

```diff
@@ -0,0 +1,9 @@
+import * as gnarkwasm from "./gnarkwasm.remote";
```

Comment:
what is this test checking, no assertions atm?

Reply:
This file removed as unused code